modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx

```diff
@@ -441,6 +441,7 @@ export default function ObjectFieldFormBase({
 					disabled={disabled}
 					emptyStateMessage={Liferay.Language.get('option-not-found')}
 					error={errors.listTypeDefinitionId}
+					id="object-field-form-base"
```

Comment:
Hi @Artartur. Please, rename the id to `objectFieldFormBase`.

Reply:
Done!